spire/middleware.py

```diff
@@ -36,7 +36,7 @@ async def dispatch(
             return Response(status_code=500, content="Internal server error")
 
         brood_user_endpoint = f"{bugout_auth_url}/user"
-        brood_user_groups_endpoint = f"{bugout_auth_url}/groups"
+        brood_user_groups_endpoint = f"{bugout_auth_url}/groups?include_metrics=false"
```

Comment:
@Andrei-Dolgolev : Can you please use the `params` key in `requests`?

https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls

Reply:
Add changes